custom-graders/v2/DemoPythonGrader/autograder/util.py

```diff
@@ -0,0 +1,82 @@
+# dependencies
+import json, sys, os
+
+# helper function to send print statements to stderr
+def print_stderr(error_msg):
+    print(str(error_msg), file=sys.stderr)
+
+# compile json object for sending score and feedback to Coursera
+def send_feedback(score, msg):
+    post = {'fractionalScore': score, 'feedback': msg}
+    with open("/shared/feedback.json", "w") as outfile:
+        json.dump(post, outfile)
+        print(json.dumps(post))
+
+# helper function to match part Ids
+def match_partId(partId, testCases):
+    # create easy-to-reference partId: funcname dictionary
+    partIds = {}
+    for key in testCases:
+        partIds[testCases[key]["partId"]] = key
+    if partId in partIds:
+        return testCases[partIds[partId]]
+    else:
+        return None
+
+
+# ---- stdout_redirected -------------------------------------------------------
+# stdout_redirected temporarily sends stdout to a temp file
```

Comment:
Francis, Now that we are using feedback.json for feedback, the grader is can just print to stdout now. Could you please make these change? Thank you.

Reply:
Updated. Thanks for the comment